pkg/security/podsecuritypolicy/seccomp/strategy.go

```diff
@@ -67,6 +68,17 @@ func NewStrategy(pspAnnotations map[string]string) Strategy {
 				allowAnyProfile = true
 				continue
 			}
+			// With the graduation of seccomp to GA we automatically convert
+			// the deprecated seccomp profile annotation `docker/default` to
+			// `runtime/default`. This means that we now have to automatically
+			// allow `runtime/default` if a user specifies `docker/default` and
+			// vice versa in a PSP.
+			if p == v1.DeprecatedSeccompProfileDockerDefault {
+				allowedProfiles[v1.SeccompProfileRuntimeDefault] = true
+			}
+			if p == v1.SeccompProfileRuntimeDefault {
+				allowedProfiles[v1.DeprecatedSeccompProfileDockerDefault] = true
+			}
```

Comment:
would the meaning be clearer like this?

```suggestion
			if p == v1.DeprecatedSeccompProfileDockerDefault || p == v1.SeccompProfileRuntimeDefault {
				allowedProfiles[v1.SeccompProfileRuntimeDefault] = true
				allowedProfiles[v1.DeprecatedSeccompProfileDockerDefault] = true
			}
```

Reply:
Probably, changed as suggested :+1: